Tests/Engine/RuleSuppression.tests.ps1

```diff
@@ -100,6 +99,34 @@ function SuppressPwdParam()
               -SuppressedOnly
             $ruleViolations.Count | Should -Be 1
         }
+
+        It "Records multiple suppressions applied to a single diagnostic" {
+            $script = @'
```

Comment:
Should we have a test for the case that there are no suppressions even though you have provide a number of them?

Reply:
I've added more test cases now